database/products.sql

```diff
@@ -0,0 +1,37 @@
+INSERT INTO `Product` (category_id, name, price, image, available) VALUES
+(1, 'Crispy Onion Rings', 6, 'Crispy-Onion-Rings.png', 1),
+(1, 'Ancho Chili Cheese Quesadilla', 6, 'Ancho-Chili-Cheese-Quesadilla.png', 1),
+(1, 'Thai Spring Rolls', 7, 'Thai-Spring-Rolls.png', 1),
+(2, 'Cream of Wild Mushroom', 4, 'Cream-of-Wild-Mushroom.png', 1),
+(2, 'Tasty Chicken Noodle soup', 4, 'Tasty-Chicken-Noodle-soup.png', 1),
```

Comment:
The default value for the `available` column is `1` so you don't need to insert it.

Reply:
Done.

Reply:
If subfolder of images not founded that is not polluting our project. but the best is to split them.
but the best done.